src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -454,19 +454,12 @@ internal virtual void ValidateParameters()
                 ThrowTerminatingError(error);
```

Comment:
@iSazonov, your last commit had 2 failures in `PowerShell-CI-linux`
Invoke-WebRequest tests.Invoke-WebRequest throws on conflicting parameters: '-ProxyCredential' and '-ProxyUseDefaultCredentials'
```powershell
Expected an exception, with FullyQualifiedErrorId 'WebCmdletProxyCredentialConflictException,,Microsoft.PowerShell.Commands.InvokeWebRequestCommand' to be thrown, but the FullyQualifiedErrorId was 'WebCmdletProxyCredentialConflictException,Microsoft.PowerShell.Commands.InvokeWebRequestCommand'. from /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1:576 char:11
    +         { Invoke-WebRequest -Uri http://httpbin.org -ProxyCredential  ?
    +           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1: line 576
576:         { Invoke-WebRequest -Uri http://httpbin.org -ProxyCredential $credential -ProxyUseDefaultCredentials } | Should -Throw -ErrorId "WebCmdletProxyCredentialConflictException,,Microsoft.PowerShell.Commands.InvokeWebRequestCommand"
```
Invoke-RestMethod tests.Invoke-RestMethod throws on conflicting parameters: '-ProxyCredential' and '-ProxyUseDefaultCredentials'
```powershell
Expected an exception, with FullyQualifiedErrorId 'WebCmdletProxyCredentialConflictException,,Microsoft.PowerShell.Commands.InvokeRestMethodCommand' to be thrown, but the FullyQualifiedErrorId was 'WebCmdletProxyCredentialConflictException,Microsoft.PowerShell.Commands.InvokeRestMethodCommand'. from /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1:2169 char:11
    +         { Invoke-RestMethod -Uri http://httpbin.org -ProxyCredential  ?
    +           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1: line 2169
2169:         { Invoke-RestMethod -Uri http://httpbin.org -ProxyCredential $credential -ProxyUseDefaultCredentials } | Should -Throw -ErrorId "WebCmdletProxyCredentialConflictException,,Microsoft.PowerShell.Commands.InvokeRestMethodCommand"
```